packages/blade/src/components/Accordion/_decisions/decisions.md

```diff
@@ -117,16 +201,52 @@ const App = () => {
 
   return (
     <Accordion expandedIndex={index} onExpandChange={onExpandChange}>
-      <AccordionItem title="Can I use a payment gateway?" description="Just use Razorpay" />
-      <AccordionItem
-        title="How can I transfer money to customers?"
-        description="Just use Razorpay"
-      />
+      <AccordionItem>
+        <AccordionItemTrigger title="How can I setup Route?" />
+        <AccordionItemBody>
+          <Text>
+            You can use Razorpay Route from the Dashboard or using APIs to transfer money to
+            customers.
+          </Text>
+        </AccordionItemBody>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionItemTrigger title="How can I setup QR Codes?" />
+        <AccordionItemBody>
+          <Text>Just use Razorpay</Text>
+        </AccordionItemBody>
+      </AccordionItem>
     </Accordion>
   );
 };
 ```
 
+#### With Individual Item
+
+```tsx
+const App = () => {
+  const [isOpen, setIsOpen] = useState(0);
+
+  return (
+    <AccordionItem
```

Comment:
I don't think we should allow this usage. It should always be wrapped within `Accordion` & the API remains consistent.